tests/GithubIntegration.py

```diff
@@ -268,10 +268,30 @@ def testGetAccessTokenWithInvalidPermissions(self):
     def testGetAccessTokenWithInvalidData(self):
         auth = github.Auth.AppAuth(APP_ID, PRIVATE_KEY)
         github_integration = github.GithubIntegration(auth=auth)
-        with self.assertRaises(github.GithubException) as raisedexp:
+
+        with self.assertRaises(github.GithubException) as raisedPermissionExp:
             github_integration.get_access_token(self.repo_installation_id, permissions="invalid_data")
+        self.assertEqual(raisedPermissionExp.exception.status, 400)
+
+        with self.assertRaises(github.GithubException) as raisedRepositoriesExp:
+            github_integration.get_access_token(self.repo_installation_id, repositories="invalid_data")
+        self.assertEqual(raisedRepositoriesExp.exception.status, 400)
+
```

Comment:
Lets also test for invalid list element types
```suggestion

        with self.assertRaises(github.GithubException) as raisedRepositoriesExp:
            github_integration.get_access_token(self.repo_installation_id, repositories=[False])
        self.assertEqual(raisedRepositoriesExp.exception.status, 400)

```